### PR TITLE
Add mention of DigiNode Tools to Raspberry Pi setup

### PR DESCRIPTION
**Pull request by saltedlolly**

### CODE REVIEW

**Comment by saltedlolly** [comment]
cc @Sbosvk

**Comment by Sbosvk** [comment]
Thank you for pinging me, Olly. I hadn't gotten notifications for this PR for some reason. Will take a look at this.

File: guides/run-full-node.md [hunk 1]
```diff
@@ -260,42 +260,52 @@ In the service file you created, replace any occurrences of `/home/digibyte/digi
...
+
+> A Raspberry Pi 4 or later is required. Raspberry Pi 3 and older are not supported. It's always recommended to have active fan cooling on the Raspberry Pi.
 {.is-info}
 
 
 ## Prerequisites
 
 | Requirements | Minimum | Recommended |
 | --- | --- | --- |
+| Hardware | Raspberry Pi 4 4Gb | Raspberry Pi 5 8Gb or better |
```
Comment (Sbosvk) [inline guides/run-full-node.md:276]: Generally speaking this is definitely reasonable. Some people like to tinker with their Raspberry Pi chips however to achieve a significantly increased throughput, enabling even older models to reach the same capacity as a Raspberry Pi 4. That said, I think it's fair to generalize here.

**Review by Sbosvk — APPROVED** [review]
Thank you, Olly, for making these changes.
It is more than fair to replace Balena with Raspberry Pi imager. In the future I will add a tabset for using `dd` as well.

**Pull request merged**